libretroshare/src/pqi/pqihandler.cc

```diff
@@ -437,43 +437,45 @@ int     pqihandler::UpdateRates()
 	RsDbg() << "UPDATE_RATES pqihandler::UpdateRates mod_index " << mod_index << " in_max_bw " << in_max_bw << " remaining in bw " << in_remaining_bw << std::endl;
 #endif
 
-	/* Allocate only half the remaining in bw, if any, to make it smoother */
-	in_max_bw = in_max_bw + in_remaining_bw / 2;
+	// allocate all remaining in bw
+	in_max_bw = in_max_bw + in_remaining_bw;
```

Comment:
That originally was made to smooth the curve of bandwidth allocation. Seconds after seconds, the available bw tends (quite rapidely) to in_max_bw+in_remaining_bw. I have the feeling that this change is what made you use the 0.95 factor below. Is that right?